torchvision/prototype/transforms/functional/_misc.py

```diff
@@ -8,7 +8,39 @@
 from torchvision.transforms import functional_tensor as _FT
 from torchvision.transforms.functional import pil_to_tensor, to_pil_image
 
-normalize_image_tensor = _FT.normalize
+
+def normalize_image_tensor(
+    image: torch.Tensor, mean: List[float], std: List[float], inplace: bool = False
+) -> torch.Tensor:
+    if not isinstance(image, torch.Tensor):
+        raise TypeError("Input img should be Tensor image")
+
+    if not image.is_floating_point():
+        raise TypeError(f"Input tensor should be a float tensor. Got {image.dtype}.")
+
+    if image.ndim < 3:
+        raise ValueError(
+            f"Expected tensor to be a tensor image of size (..., C, H, W). Got tensor.size() = {image.size()}"
+        )
+
+    if (isinstance(std, (tuple, list)) and not all(std)) or std == 0:
```

Comment:
Do we need the first part of the check? What input would fail `isinstance(std, (tuple, list))`? Do we actually allow scalars here? Otherwise, this should be sufficient

```suggestion
    if not all(std):
```

Reply:
We actually allow scalars. It's not visible due to the JIT-script types but if you pass `mean=0.5, std=0.5` it works. So I'm keeping this for BC and provide separate benchmarks.

Reply:
Good catch. I also had to rewrite the check because JIT couldn't understand the assertions were correct in one line... This version seems to pass. I've updated the benchmarks and we are still good.